.github/actions/build/action.yml

```diff
@@ -0,0 +1,28 @@
+name: build-wheel
+description: "Build a wheel"
+
+outputs:
+  hashes:
+    description: "Hashes of the dist directory"
+    value: ${{ steps.hashes.outputs.hashes }}
+
+runs:
+  using: composite
+
+  steps:
+    - name: Build wheel
+      shell: bash
+      run: SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct) pixi run build
+
+    - name: "Generate hashes"
+      id: hashes
+      shell: bash
+      run: cd dist && echo "hashes=$(sha256sum * | base64 -w0)" >> $GITHUB_OUTPUT
+
+    - name: "Upload dists"
+      uses: "actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02"
+      with:
+        name: "dist"
+        path: "dist/"
+        if-no-files-found: error
+        retention-days: 5
```

Comment:
Why are we upload the package data to GH if it gets uploaded to PyPI anyway? What's the benefit of having this here over downloading from PyPI if required?